minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/RunMiNiFi.java

```diff
@@ -275,7 +275,7 @@ public static void main(String[] args) throws IOException, InterruptedException
         }
     }
 
-    private static File getBootstrapConfFile() {
+    public static File getBootstrapConfFile() {
```

Comment:
I changed this to `public` so the new `WindowsService` class could call it instead of duplicating the function.